cumulus_etl/errors.py

```diff
@@ -56,5 +58,5 @@ class FatalError(Exception):
 
 def fatal(message: str, status: int) -> NoReturn:
     """Convenience method to exit the program with a user-friendly error message a test-friendly status code"""
-    print(message, file=sys.stderr)
+    rich.console.Console(stderr=True).print(message, style="bold red", highlight=False)
```

Comment:
I tested this red config (which I stole from an example in `rich`'s docs on printing to the error console) on several different terminal color schemes -- seemed decent to me. Here's an example:

![Screenshot from 2024-03-18 15-29-56](https://github.com/smart-on-fhir/cumulus-etl/assets/1196901/4c030547-9dda-4e31-8f6b-571989d25bdd)